src/python/pants/backend/java/compile/javac_subsystem.py

```diff
@@ -23,5 +23,8 @@ def register_options(cls, register):
             advanced=True,
             help="The JDK to use for invoking javac."
             " This string will be passed directly to Coursier's `--jvm` parameter."
-            " Run `cs java --available` to see a list of available JVM versions on your platform.",
+            " Run `cs java --available` to see a list of available JVM versions on your platform."
+            " If the string 'system' is passed, Coursier's `--system-jvm` option will be used"
```

Comment:
Formatting nit, good to add `\n\n` after this first sentence so that they're separate paragraphs. We've been doing that a lot more recently to make our help messages more readable.

Reply:
Another nit: can wrap parens around the multi-line string to encourage `black` to format it more nicely: `help=("The JDK ...")`.

Reply:
Done